packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -248,15 +248,6 @@ enum _LogType {
 }
 
 class AnsiTerminal {
-  AnsiTerminal() {
-    if (platform.isWindows) {
-      supportsColor = platform.ansiSupported;
-    } else {
-      final String term = platform.environment['TERM'];
-      supportsColor = (term != null && term != 'dumb');
```

Comment:
According to the Dart Docs it shouldn't: https://api.dartlang.org/dev/1.23.0-dev.11.5/dart-io/Stdout/supportsAnsiEscapes.html